src/queue.c

```diff
@@ -8142,8 +8151,10 @@ DISPATCH_EXPORT DISPATCH_NOTHROW
 void
 libdispatch_init(void)
 {
+#if 0
```

Comment:
Similarly here?

Reply:
This assertion is actually being tripped currently.

Reply:
I think that this is a problem with the locks shims.